docs/configuration.md

```diff
@@ -37,12 +43,96 @@ If you are running these applications without Docker, just make sure that any en
+- **SERVER_ENV_FILE** The name of an environment file to be passed into the API Server container by docker compose. Defaults to ".env" if left blank. Used especially for building a "test" version of the project for end-to-end testing.
+- **SERVER_LOG_LEVEL** Used by Winston.js in the API server to determine how much logging to output. Reasonable values are "debug", "info", and "error". Defaults to "info".
+
+### Database
+
+- **READ_ONLY_DATABASE_URL** (optional) Database replica for reads.
+- **POSTGRES_DB** database name (e.g. "polis-dev")
+- **POSTGRES_HOST** database host (e.g. postgres:5432 if using docker compose, localhost:5432 if using local db)
+- **POSTGRES_PASSWORD** database password
+- **POSTGRES_PORT** typically 5432
+- **POSTGRES_USER** typically "postgres". Any username will be used by the docker container to create a db user.
+- **DATABASE_URL** should be the combination of above values, `postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}/${POSTGRES_DB}`
+
+### Docker Concerns
+
+- **TAG** used by **COMPOSE_PROJECT_NAME** below. Defaults to "dev".
+- **COMPOSE_PROJECT_NAME** Used by docker compose to label containers and volumes. Useful in development if you are (re)-building and deleting groups of docker assets.
+
+### Ports
+
+- **API_SERVER_PORT** typically 5000. Used internally within a docker network and/or behind a proxy.
+- **HTTP_PORT** typically 80. Port exposed by Nginx reverse proxy.
+- **HTTPS_PORT** typically 443. Port exposed by Nginx reverse proxy.
+- **STATIC_FILES_PORT** typically 8080. Used internally within a docker network and/or behind a proxy.
+- **STATIC_FILES_ADMIN_PORT** same as **STATIC_FILES_PORT** unless you are hosting client-admin separately from file-server. Useful in local development.
+- **STATIC_FILES_CLIENT_PORT** same as **STATIC_FILES_PORT** unless you are hosting client-participation separately from file-server. Useful in local development.
+
+### Email Addresses
+
+- **ADMIN_EMAIL_DATA_EXPORT** email address from which data export emails are sent.
+- **ADMIN_EMAIL_DATA_EXPORT_TEST** email address to receive periodic export test results, if configured below.
+- **ADMIN_EMAIL_EMAIL_TEST** email address to receive backup email system test.
+- **ADMIN_EMAILS** array of email addresses to receive team notifications.
+- **POLIS_FROM_ADDRESS** email address from which other emails are sent.
+
+### Boolean Flags
+
+(All can be left blank, or `false`)
+
+- **BACKFILL_COMMENT_LANG_DETECTION** Set to `true`, if Comment Translation was enabled, to instruct the server upon the next initialization (reboot) to backfill detected language of stored comments. Default `false`.
+- **CACHE_MATH_RESULTS** Set this to `true` to instruct the API server to use LRU caching for results from the math service. Default `false`.
```

Comment:
I think this one should default to true. Was this not the case previously?

Reply:
Previously (see `stable`) there was no mention of or reference to this variable anywhere other than a single line of code that checked it (server.ts)

`  let pcaCacheSize = process.env.CACHE_MATH_RESULTS === "true" ? 300 : 1;`

So I would infer the default to be falsey.

Reply:
Yeah, I came across this earlier and can verify that we explicitly set to `true` in the prod env variables. However, I don't think there's any good reason to have this set to `false` by default, except maybe in development (but even there, matching `prod` behavior seems more sound generally).